README.md

```diff
@@ -99,10 +100,11 @@ The Operations Team takes care of legal documents, invoicing and remittances.
 
 - [Kasper Mai Jørgensen](https://github.com/KasperJor)
 - [Rouven Pérez](https://github.com/RouvenP)
+- [Federica Dubbini](https://github.com/fededubbi)
 
 ## :pencil: Process
 
-> **:loudspeaker:** The process below allows for grants of up to **$30,000 for first-time applications** and **$100,000 for follow-up ones**. Payment is made in Bitcoin, USDT or DAI. If you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding above the aforementioned limits**, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). Note that this is generally a slower process and imposes stricter requirements on applicants.
+> **:loudspeaker:** The process below is independent of the [level](#level_slider-level). Payment is made in Bitcoin, USDT or DAI. If you want to apply in **private**, need to be paid out in **fiat** and/or you **seek funding above the aforementioned limits**, you can apply [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). Note that this is generally a slower process and imposes stricter requirements on applicants.
```

Comment:
> seek funding above the aforementioned limits

I don't think this is applicable anymore?

Reply:
I removed it